src/SensorVisuals/components/VisualsByCode/VisualsByCode.js

```diff
@@ -246,9 +247,19 @@ const VisualsByCode = () => {
                 size="small"
                 variant="contained"
                 color="primary"
-                onClick={() => setShowIssueDialog(!showIssueDialog)}
+                onClick={() => {
+                  dispatch(
+                    setIssueDialogData({
+                      nickname: user.nickname,
+                      dataType: 'json',
+                      defaultText:
+                        'Make sure to include treatment, rep, which sensor(s)/depth(s) are problematic; etc. Include as much detail as possible.',
+                      setShowNewIssueDialog: true,
+                    }),
+                  );
+                }}
               >
-                {showIssueDialog ? 'Cancel' : 'Create comment'}
+                {issueDialogData.setShowNewIssueDialog ? 'Cancel' : 'Create comment'}
```

Comment:
there's a weird bug in this file where it added an extra cancel button and the top one is broken
![canc](https://user-images.githubusercontent.com/55995241/183123053-2e050fdb-8886-46e3-9817-b21effd831cc.PNG)
n